Allow livereload to watch files in all theme subdirectories

sanduhrs opened the pull request:
Livereload per default watches the following file extensions: 'html,css,js,png,gif,jpg,php,php5,py,rb,erb,coffee'
But is configured to only watch the css directory.

It might make sense to just watch all the theme's subdirectories, including css, js, images.
Also adding the -d flag for more verbosity.

sanduhrs committed:
Allow lovereload to watch files in all theme subdirectories instead of just css add dgb flag

sanduhrs committed:
Fix theme generator test

Chi-teck commented:
> html,css,js,png,gif,jpg,php,php5,py,rb,erb,coffee

But only "css,js,png,gif,jpg" make sense for Drupal themes. Don't it? Is it a good idea to broadcast changes in twig and theme files as well? 
Also I think it should not watch files in `node_modules` directory.

> Also adding the -d flag for more verbosity.

Is this information really valuable?

sanduhrs committed:
Add livereload folder exclusion and file types to watch

sanduhrs commented:
I removed the -d flag. Thought it would be nice for the first run to check if things are working, but can add this manually.

Adjusted the file types based on the following list:
https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types

Added an exclusion for node_modules.

Chi-teck commented:
Thank you.
